src/middlewared/middlewared/plugins/network.py

```diff
@@ -615,6 +624,7 @@ def iface_extend(self, iface_state, configs, is_freenas, fake=False):
         the interfaces changes happened as planned from the user. If checkin does not happen
```

Comment:
With `rollback` true (default), changes will be rolled back if they have not been applied.
Network changes are rolled back after `checkin_timeout` (time in seconds) passes without
the user committing them.